generated/nidigital/nidigital/session.py

```diff
@@ -3090,10 +3632,13 @@ def _load_levels(self, file_path):
     def load_pattern(self, file_path):
         r'''load_pattern
 
-        TBD
+        Loads the specified pattern file.
 
         Args:
-            file_path (str):
+            file_path (str): Absolute file path of the binary .digipat pattern file to load. Specify the pattern to burst using the ConfigureStartLabel method or the **startLabel** parameter of the _burst_pattern method.
```

Comment:
"Specify the pattern to burst using the ConfigureStartLabel() method"

ConfigureStartLabel() method --> start_label property

Reply:
Done